python/tvm/testing/aot.py

```diff
@@ -765,6 +793,8 @@ def run_and_check_body(base_path):
                 )
 
         use_usmp = runner.pass_config.get("tir.usmp.enable", False)
+        options = runner.pass_config.get("relay.ext.cmsisnn.options")
+        debug_last_error = options.get("debug_last_error", False) if options else False
```

Comment:
The AOT test runner shouldn't have knowledge of `CMSIS-NN` specific arguments, can we pass `debug_last_error` as a parameter?

Reply:
Done. Thanks for the review!